identity/src/main/java/com/stripe/android/identity/navigation/UploadDestinations.kt

```diff
@@ -1,95 +1,199 @@
 package com.stripe.android.identity.navigation
 
-import androidx.core.os.bundleOf
+import androidx.annotation.StringRes
+import androidx.navigation.NavBackStackEntry
+import androidx.navigation.NavType
+import androidx.navigation.navArgument
 import com.stripe.android.identity.R
+import com.stripe.android.identity.networking.models.CollectedDataParam
+import com.stripe.android.identity.states.IdentityScanState
 
-internal class PassportUploadDestination(
+internal abstract class DocumentUploadDestination(
```

Comment:
DocumentUploadDestination is the super class of `PassportUploadDestination `, `IDUploadDestination` and `DriverLicenseUploadDestination`, as their parameters list are exactly the same